### PR TITLE
[overlay] Add ModelIO overlay

### PR DESCRIPTION
**Pull request by moiseev**

### CODE REVIEW

**Comment by moiseev** [comment]
@swift-ci Please test

File: apinotes/ModelIO.apinotes [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
```
Comment (jrose-apple) [inline apinotes/ModelIO.apinotes]: No need to add the API notes; those are part of the framework now.

Comment (moiseev) [inline apinotes/ModelIO.apinotes]: They are not. Not yet, anyway.

Comment (jrose-apple) [inline apinotes/ModelIO.apinotes]: Please don't add them here. We're about to get rid of all the existing ones, and it's just a coincidence that they don't have any yet.

Comment (moiseev) [inline apinotes/ModelIO.apinotes]: They will ship apinotes eventually in the Framework, meanwhile including them allows us to merge this change here, and include it in the previews.

**Comment by moiseev** [comment]
@swift-ci Please test

**Comment by moiseev** [comment]
@swift-ci Please test

**Pull request merged**